queries/ec2/no_broad_public_ingress_on_port_3389.sql

```diff
@@ -11,5 +11,7 @@ SELECT account_id,
        'instance should not allow access to port 3389 from the entire internet' as cq_reason
 FROM aws_security_group_ingress_rules
 WHERE (ip = '0.0.0.0/0' OR ip = '::/0')
-    AND (from_port IS NULL AND to_port IS NULL) -- all prots
-    OR 3389 BETWEEN from_port AND to_port
+    AND (
+       (from_port IS NULL AND to_port IS NULL) -- all prots
```

Comment:
```suggestion
       (from_port IS NULL AND to_port IS NULL) -- all ports
```

Same as https://github.com/cloudquery-policies/aws/pull/160#discussion_r907070133